charts/apps/kubeflow-pipelines/rds-only/crds/clusterworkflowtemplates.argoproj.io-CustomResourceDefinition.yaml

```diff
@@ -1,3 +1,4 @@
+2023/03/17 11:44:54 well-defined vars that were never replaced: kfp-app-name,kfp-app-version
```

Comment:
bug in the helmify?

Reply:
mannually removed. This will resolve after we use kustomize v5.0.1

Reply:
please use 5.0.1 because there are multiple files where this is happening

Reply:
yea. All were fixed.